README.md

```diff
@@ -23,8 +23,8 @@ All supported boards mentioned above are set up to use bootloader with a CI test
 To either use a different board or a different SD card slot, you must rebuild the bootloader and update the application.
 To do this:
 
-1. Create a bootloader for your board in the project [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader).
-2. Update your board to use the newly created booloader image. To do this, set the target value `bootloader_img` to the file path of the bootloader image.
+1. Create a bootloader for your board in the project [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader). The bootloader imagine is named ``mbed-os-example-bootloader.bin``.
+2. Update your board to use the newly created bootloader image. To do this, set the target value `bootloader_img` to the file path of the bootloader image.
 
 ```
     "target_overrides": {
```

Comment:
Current: "NUCLEO_F429ZI": {
                       "target.bootloader_img": "bootloader/NUCLEO_F429ZI.bin"

Proposed: In mbed_app.json, to add your newly built bootloader image to the application, make the following changes:
"<target_name>": {
   "target.bootloader_img": "bootloader\<Your_new_bootloader_filename).img

Something like this?? Just to keep it more generic.. else in the future we will have a phase similar to the era of K64F .. but for the NUCLEO_F429ZI this time.. what do you think?